src/components/InventoryPage/ImageModal.jsx

```diff
@@ -1,13 +1,44 @@
 import React from 'react';
+import './InventoryPage.module.css';
+import {
+  ModalOverlay,
+  ModalContent,
+  ModalBody,
+  ModalCloseButton,
+  Textarea,
+  Modal,
+  Image,
+} from '@chakra-ui/react';
+import { PropTypes } from 'prop-types';
 
-// import { Textarea } from '@chakra-ui/react';
-const ImageModal = () => {
+const ImageModal = ({ isOpenImageModal, onCloseImageModal, image }) => {
+  const { imageUrl, notes } = image;
   return (
     <>
-      {/* <img src="" alt="" />
-      <Textarea /> */}
+      <Modal isOpen={isOpenImageModal} onClose={onCloseImageModal} size="full">
+        <ModalOverlay />
+        <ModalContent>
+          <ModalCloseButton />
+          <ModalBody>
+            <Image h={500} w="35%" my={50} mx="auto" src={imageUrl} />
+            <Textarea placeholder={notes} isDisabled />
```

Comment:
Since we can't edit this text and disabling makes it hard to read, let's use a Chakra [Text](https://chakra-ui.com/docs/components/text) component.